app/assets/javascripts/initializers/initializeFetchFollowedArticles.js.erb

```diff
@@ -190,10 +190,9 @@ function intersect_arrays(a, b) {
 }
 
 function insertTimes() {
-  var els = document.getElementsByClassName('time-ago-indicator-initial-placeholder');
-
-  for (var i = 0; i < els.length; i++) {
-    var el = els[i];
-    el.outerHTML = timeAgo(el.dataset.seconds);
+  var elements = document.getElementsByClassName('time-ago-indicator-initial-placeholder');
+  for (var i = 0; i < elements.length; i++) {
+    var element = elements[0];
```

Comment:
aren't you getting out the same exact element each iteration here?

Reply:
No I'm not because `getElementsByClassName` returns a Live HTMLCollection meaning that it will change as we change the DOM

Reply:
So `elements[0]` will produce the next unchanged element every iteration.

Reply:
ok got it! thanks!

btw I'd add a comment before this line explaining just that, because it instinctively looks like a bug :D

Reply:
Yes I think it is required to clear the confusion. Thanks !